onnx/numpy_helper.py

```diff
@@ -277,8 +274,10 @@ def from_array(array: np.ndarray, /, name: str | None = None) -> onnx.TensorProt
     }:
         # Pack the array into int4
         array = _pack_4bitx2(array)
-    if not _IS_LITTLE_ENDIAN:
-        array = array.view(array.dtype.newbyteorder("<"))
+
+    if sys.byteorder == "big":
+        # Convert endian from big to little
+        array = array.byteswap()
```

Comment:
I wonder if generalizing the logic into

tobytes_little_endian(array) -> bytes

would make this more consistent?